manifests/init.pp

```diff
@@ -95,7 +101,9 @@
   String  $config_file            = $telegraf::params::config_file,
   String  $config_file_owner      = $telegraf::params::config_file_owner,
   String  $config_file_group      = $telegraf::params::config_file_group,
+  String  $config_file_mode       = $telegraf::params::config_file_mode,
```

Comment:
can you please use the filemode datatype? https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/types/filemode.pp

Reply:
updated